spec/system/index_spec.rb

```diff
@@ -83,22 +83,44 @@
   end
 end
 
-describe 'searching for a bus stop by route' do
+describe 'searching for a bus stop by sequence' do
   let(:user) { create :user }
   let!(:route) { create :route }
-  let!(:bus_stop) { create :bus_stop, routes: [route] }
+  let!(:bus_stop) { create :bus_stop }
+  let!(:bus_stops_route) { create :bus_stops_route, bus_stop: bus_stop, route: route }
   before :each do
     when_current_user_is user
     visit root_url
   end
   context 'route from the dropdown' do
-    it 'redirects to bus stops by route page' do
+    it 'redirects to bus stops by sequence' do
       within 'form', text: 'Select a route' do
         select route.number, from: 'Select a route'
```

Comment:
This is ambiguous, view needs to be corrected.

Reply:
:(

Reply:
@jforgue is fixing the view while I'm fixing the test